src/doc/en/api/class/HttpService.md

```diff
@@ -1,4 +1,53 @@
+# Summary
+Allows a server to make HTTP requests.
+
 # Details
+The URL schemes allowed by HttpService are `http` and `https`. Domains are
+filtered using a blacklist. Generally, requests to any URL are allowed, while
+requests to Roblox domains are disallowed. If a URL is disallowed or otherwise
+invalid, then the following error will be thrown:
+
+	URL: Trust check failed
+
+Where `URL` is the given URL.
+
+If the URL is empty, then the following error will be thrown:
+
+	Empty URL
+
```

Comment:
Errors might require their own section. RequestAsync returns a different error when trust check fails, and http can return a variety of errors. Not sure if documenting these is out of scope.

There appears to be a general `HttpError` prefix that captures protocol errors, though the exact content is almost certainly unstandardized and therefore volatile.